core/src/main/java/com/yahoo/oak/EntrySet.java

```diff
@@ -32,83 +30,48 @@
  * -----------------------------------------------------------------------------|
  * 3 | NOT IN USE:             saved for alignment and future usage             |
  * --------------------------------------------------------------------------------------
- * 4 | KEY_REFERENCE          | these 2 integers together, represented as long  | entry with
- * --|                        | provide KEY_REFERENCE. Pay attention that       | entry index
- * 5 |                        | KEY_REFERENCE is different than VALUE_REFERENCE | 1
+ * 4 | KEY_REF_OFFSET          | these 2 integers together, represented as long  | entry with
```

Comment:
Why the change to "ref_offset"? We call it a reference everywhere else. It implies including all the information to access the key/value. This includes the offset.

Reply:
fixed the comment